### ISSUE TITLE
node_modules/require-dir error

### ISSUE THREAD

**Issue by katiewelton** [Shopify/slate#170]
None of `slate watch` `slate start` etc are working. I'm getting this error:

```
/Users/Katie/Desktop/feather-edge-theme/node_modules/require-dir/index.js:97
    if (!require.extensions.hasOwnProperty(ext)) {
                                    ^

TypeError: require.extensions.hasOwnProperty is not a function
    at requireDir (/Users/Katie/Desktop/feather-edge-theme/node_modules/require-dir/index.js:97:37)
    at Object.<anonymous> (/Users/Katie/Desktop/feather-edge-theme/node_modules/@shopify/slate-tools/lib/gulpfile.js:38:23)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Liftoff.handleArguments (/Users/Katie/Desktop/feather-edge-theme/node_modules/gulp/bin/gulp.js:116:3)
```

I have made sure I am running the most up to date versions of npm, node and slate. I have updated all of the node modules as well.

Funny thing is this was working fine on Friday. I've come back to work on Monday and suddenly this? 

Is this a problem with the module itself, or is there something I can do? Thanks!!!

**Comment by justinhough** [comment]
Having the same issue as well with any new Slate installs?

**Comment by justinhough** [comment]
The issue is with the requireDir there is an issue that was fixed with `0.3.1`. It's documented here: https://github.com/aseemk/requireDir/issues/45. I manually upgraded my local install of requireDir to `0.3.2` and it resolved the issues I was having.

**Comment by ry167** [comment]
I used `nvm` to downgrade to `v7.10.0` to solve this
